app/src/androidTest/java/com/sdp/swiftwallet/presentation/signIn/ForgotPwActivityTest.java

```diff
@@ -85,7 +85,19 @@ public void no_email_raises_error() {
 
     onView(withId(R.id.emailField)).check(matches(hasFocus()));
   }
-  
+
+  @Test
+  public void firesCorrectIntentAfterLinkSent(){
+    onView(withId(R.id.emailField)).perform(typeText("dummy@gmail.com"), closeSoftKeyboard());
+    onView(withId(R.id.sendReset)).perform(click());
+
+    // Avoid softban reset with firebase..
+    /*intended(allOf(
+            toPackage("com.sdp.swiftwallet"),
+            hasComponent(hasClassName(LoginActivity.class.getName()))
+    ));*/
```

Comment:
Is it not possible to uncomment the intended part now that the email is changed ?